app/views/events/_event.html.erb

```diff
@@ -4,4 +4,5 @@
   <div><span class="font-bold	text-lg">Event date:</span> <%= event.date %></div>
   <div><span class="font-bold	text-lg">Event location:</span> <%= event.location %></div>
   <div><span class="font-bold	text-lg">Event price:</span> <%= event.price %></div>
+  <div><%= render_event_image_if_exists event %></div>
```

Comment:
I understand that you're evaluating that the image exists, right?
There's another way to render the event image if exist.

event.image.attached? if true, then you can render the image otherwise do nothing.

Did you tried this way?
If use this way, then the helpers are not necessary any more.

Reply:
Yeah. That helper used to evaluate (through the now named image_is_saved_and_exists? method inside Event model) if the image existed before the rendering; now it just calls the image_tag helper, so the condition was moved to the corresponding ERB views.